tcms/core/tests/test_ajax.py

```diff
@@ -5,14 +5,13 @@
 from django import test
 from django.urls import reverse
 from django.conf import settings
+from django.db.models import Count
 
-from tcms.testcases.models import TestCase
+from tcms.testcases.models import TestCase, TestCaseTag
 from tcms.tests import BasePlanCase
+from tcms.core.ajax import _TagCounter
```

Comment:
Move that above the imports of TestCase and TestCaseTag. Tests for the most part follow the convention to import classes and functions from  the Python standard library first, then from Django and various other dependencies, then tcms and finally all testing related classes.